src/script/page/MainContent/MainContent.test.tsx

```diff
@@ -32,20 +33,22 @@ jest.mock(
     },
 );
 
-// TODO: Remove after all migrations
-jest.mock('src/script/page/RightSidebar/utils/toggleRightPanel', () => () => jest.fn());
-
 describe('Preferences', () => {
   const defaultParams = {
     contentViewModel: {
       repositories: {} as any,
       state: ko.observable(ContentState.PREFERENCES_ACCOUNT),
     } as ContentViewModel,
+    openRightSidebar: jest.fn(),
   };
 
   it('renders the right component according to view state', () => {
     jest.useFakeTimers();
-    const {queryByText, getByText} = render(<MainContent {...defaultParams} />);
+    const {queryByText, getByText} = render(
+      <StyledApp themeId={THEME_ID.DEFAULT}>
```

Comment:
We have a util called `withTheme` in `TestUtil.tsx" you can use

Reply:
ooohhh nice catch 👍